common/api-review/util.api.md

```diff
@@ -482,6 +482,9 @@ export interface Subscribe<T> {
 // @public (undocumented)
 export type Unsubscribe = () => void;
 
+// @public
+export function updateEmulatorBanner(name: string, isRunningEmulator: boolean): void;
```

Comment:
does this need to be public? or should it be prefix with _?

Reply:
It's okay to have this public, it's an internal package